client/js/store/modules/auth.js

```diff
@@ -67,12 +67,11 @@ export default {
         );
       });
     },
-
     requestLogout(context) {
       return new Promise((resolve, reject) => {
-        io.socket.get('/user/logout', {}, function handleResponse(resData, jwres) {
+        io.socket.get('/user/logout', {}, async function handleResponse(resData, jwres) {
```

Comment:
```suggestion
        io.socket.get('/user/logout', {}, function handleResponse(resData, jwres) {
```
Oops, don't need async here.